Makefile

```diff
@@ -51,7 +51,7 @@ POSTGRESQL_CONTROLLER_INTEGRATION_HOST_CONTAINER=postgresql-controler-int-test
 .PHONY: test/integration/postgresql/run
 test/integration/postgresql/run:
 	@echo Running integration test PostgreSQL instance on localhost:5432:
-	-docker run --rm -p 5432:5432 -e POSTGRES_USER=iam_creator --name ${POSTGRESQL_CONTROLLER_INTEGRATION_HOST_CONTAINER} -d timms/postgres-logging:11.5 && sleep 5
+	-docker run --rm -p 5432:5432 -e POSTGRES_USER=admin --name ${POSTGRESQL_CONTROLLER_INTEGRATION_HOST_CONTAINER} -d timms/postgres-logging:11.5 && sleep 5 && echo "CREATE USER iam_creator WITH CREATEDB CREATEROLE VALID UNTIL 'infinity';" | psql -hlocalhost -Uadmin -d postgres
```

Comment:
This is closer to the real scenario on RDS. The `iam_creator` is just a role with `CREATEDB` and `CREATEROLE` privileges instead of `superuser`. 

With this change the bug appears locally as well.